web/packages/shared/components/AccessRequests/NewRequest/RequestCheckout/RequestCheckout.tsx

```diff
@@ -623,6 +627,29 @@ function TextBox({
   );
 }
 
+function getPrettyResourceKind(kind: ResourceKind): string {
+  switch (kind) {
+    case 'role':
+      return 'Role';
+    case 'app':
+      return 'Application';
+    case 'node':
+      return 'Server';
+    case 'resource':
+      return 'Resource';
+    case 'db':
+      return 'Database';
+    case 'kube_cluster':
+      return 'Kubernetes';
+    case 'user_group':
+      return 'User Group';
+    case 'windows_desktop':
+      return 'Desktop';
+    default:
+      kind satisfies never;
```

Comment:
```suggestion
      kind satisfies never;
      return kind;
```

In this scenario, I think it's hard to say what the source of truth for those kinds is and if it's always going to be in sync with types. Since this function returns a value that's meant for display purposes only, we can return regular `kind` to be safe.